assets/scss/_base.scss

```
@@ -77,6 +77,13 @@ footer {
     font-size: 1rem;
     border: 0px;
   }
+  
+  .button:hover {
+
+    color: white;
+    background-color:#298af5;
+
+  }
```

I would then put this pseudo class inside the button class above
```suggestion
    :hover {
      color: white;
      background-color:#298af5;
  }
```